# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../zeroinstall/dbus-api.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur zero-install-0.1.27-orig/control.c zero-install-0.1.27/control.c
--- zero-install-0.1.27-orig/control.c	2004-08-22 14:00:56.000000000 +0000
+++ zero-install-0.1.27/control.c	2006-04-23 18:29:23.352659750 +0000
@@ -257,7 +257,7 @@
 				     void           *user_data)
 {
 	if (dbus_message_is_signal(message,
-				DBUS_INTERFACE_ORG_FREEDESKTOP_LOCAL,
+				DBUS_INTERFACE_LOCAL,
 				"Disconnected"))
 	{
 		if (list_contains(&monitors, connection))
